app-foundation/3-networks-extension/README.md

```diff
@@ -0,0 +1,26 @@
+# 3-networks-extension
+
+This is an additional Terraform configuration for [example-foundation 3-Networks](https://github.com/terraform-google-modules/terraform-example-foundation/tree/master/3-networks) and is used to set up the subnets, additional firewall rules for the Bank of Anthos example application across dev/non-prod/prod environments.
+
+## Prerequisites
+
+1. 0-bootstrap executed successfully.
+1. 1-org executed successfully.
+1. 2-environments/envs/ executed successfully.
+
+## Order of Execution
```

Comment:
Lets make sure to call out that this expects 3-networks to be in app-foundation. in the pre req. In the main readme we should be explicit that it should be cloned into app-foundation for the purposes of this demo.